### ISSUE TITLE
Remove stderr logging code, if there are no consumers

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#6932]
In #6922 we asked Ops to disable logging to stderr to see if anybody is actually consuming those logs. Once train-130 rolls out, and if nobody complains their logs are missing, let's remove the stderr logging code.

ref #6918

**Comment by jrgm** [comment]
In light of other events, is this issue a moot point?

**Comment by shane-tomlinson** [comment]
> In light of other events, is this issue a moot point?

No, that's why I filed #6956. I'd still like to be able to disable the StdErrCollector w/o disabling flow and amplitude events.
